frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -279,3 +279,61 @@ i.e.
     }
   }
 }
+
+// Footer
```

Comment:
This took a lot of custom CSS. There aren't a lot of settings options for customizing the Footer and Identifier. Especially wrt layout and color.